src/App.vue

```diff
@@ -158,21 +157,16 @@ import { useStore } from '@/store/store'
     SbcFooter,
     ...Dialogs,
     ...Views
-  },
-  mixins: [
-    CommonMixin,
-    FilingTemplateMixin
-  ]
+  }
 })
-export default class App extends Vue {
+export default class App extends Mixins(CommonMixin, FilingTemplateMixin) {
```

Comment:
I changed over all mixins like this -- now the mixin properties/methods are visible in this component (including typings), so you F12 them to go to their declaration instead of seeing an error 🎉